### DISCUSSION TITLE
Neovim Copilot error on close buffer after upgrade to Neovim v0.9.5

### DISCUSSION THREAD

**Discussion by starsinmypockets** [community/community#108384] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

After upgrading to Neovim v0.9.5 I see an error whenever closing a buffer:

```
Error detected while processing BufUnload Autocommands for "*"..function <SNR>41_Event[5]..BufUnload Autocommands for "*"..function <SNR>41_Event:                                            
line    4:                                                                                                                                                                                    
E118: Too many arguments for function: copilot#logger#Exception 
```
copilot.vim is at commit = '315c6d2b16e018cb8020f20aaa7081ebc4070828'

I am able to authenticate and copilot works as expected, except for this error on close buffer.

**Comment by tpope** [comment]
Version 1.18.0 (that's the version from the commit) has a `copilot#logger#Exception` that accepts multiple arguments, so you shouldn't be seeing an error like that. Maybe something is wrong with your install? Please try uninstalling and reinstalling and see if that helps.

**Comment by starsinmypockets** [comment]
Tried reinstalling. Also ran PlugUpdate. Same error.

**Reply by tpope** [reply]
Do you have 2 versions of the plugin installed? Mismatched versions of the files installed is the only way I can conceive of this happening.

**Reply by starsinmypockets** [reply]
Hmm. Maybe I installed this via another method. I'll look into that. Thanks for your help!

**Reply by starsinmypockets** [reply]
The duplicate plugin theory doesn't totally make sense because if I uninstall via Plug the plugin is disabled and the error goes away.

**Reply by tpope** [reply]
It makes sense if the duplicate installation is missing some files. Try `:call copilot#logger#Exception('test')` when the plugin is not installed. What happens?

**Comment by starsinmypockets** [comment]
OK yeah it still gives the exception. Hmmm. I guess I have to figure out where this is getting loaded. Thanks again for your help.

**Reply by tpope** [reply, marked as answer]
The answer is in `:scriptnames`. Or `:echo findfile('autoload/copilot/logger.vim', &runtimepath)` should work.

**Reply by omerfyalcin** [reply]
I had the same issue and what you suggested did work for me too. After `:echo findfile('autoload/copilot/logger.vim', &runtimepath)`  I removed the entire /github/ directory under which the `logger.vim` was and then reinstalled the plugin. It works fine now. Thank you!

**Comment by starsinmypockets** [comment]
Yes -- that gets the goods!  Thanks again sir.
